Cargo.toml

```
@@ -24,12 +24,16 @@ bevy_ecs_macros = { version = "0.16.0" }
 bevy_reflect = { version = "0.16.0" }
 bevy_utils = { version = "0.16.0" }
 bevy_internal = { version = "0.16.0", features = ["track_location"] }
-dioxus-devtools = { version = "0.7.0-alpha.0" }
-bevy_simple_subsecond_system_macros = { path = "macros", version = "0.1.7" }
+dioxus-devtools = { version = "0.7.0-alpha.0", git = "https://github.com/DioxusLabs/dioxus.git", rev = "b2bd1f48d434becfff9f0ea390ad8ef46996300a" }
```

```suggestion
dioxus-devtools = { version = "0.7.0-alpha.0", git = "https://github.com/DioxusLabs/dioxus.git", rev = "b2bd1f" }
```
I missed this but it works here too!